client/src/App.js

```diff
@@ -2,20 +2,45 @@ import React, { Component } from 'react';
 import './App.css';
 import NavBar from './navbar';
 import ApartmentMap from './apartmentmap';
+import NodeGeocoder from 'geocoder';
 
 class App extends Component {
-  componentDidMount(){
-    // fetch('api/vi/apartments')
-  }
   state = {
     markers: [{
       position: {
         lat: 39.7392, lng: -104.9903
       },
-      key: `Taiwan`,
+      key: `Denver`,
       defaultAnimation: 2,
     }],
```

Comment:
Let's try initializing state as `state={ markers: [] }`. Probably better to show a flash of un-styled content then a flash of a mysterious marker that doesn't mark a campaign.